xbmc/pvr/recordings/PVRRecordingsPath.cpp

```diff
@@ -27,21 +27,25 @@
 
 using namespace PVR;
 
-const std::string CPVRRecordingsPath::PATH_RECORDINGS         = "pvr://recordings/";
-const std::string CPVRRecordingsPath::PATH_ACTIVE_RECORDINGS  = "pvr://recordings/active/";
-const std::string CPVRRecordingsPath::PATH_DELETED_RECORDINGS = "pvr://recordings/deleted/";
+const std::string CPVRRecordingsPath::PATH_RECORDINGS               = "pvr://recordings/";
+const std::string CPVRRecordingsPath::PATH_ACTIVE_TV_RECORDINGS     = "pvr://recordings/active/tv/";
+const std::string CPVRRecordingsPath::PATH_ACTIVE_RADIO_RECORDINGS  = "pvr://recordings/active/radio/";
+const std::string CPVRRecordingsPath::PATH_DELETED_TV_RECORDINGS    = "pvr://recordings/deleted/tv/";
+const std::string CPVRRecordingsPath::PATH_DELETED_RADIO_RECORDINGS = "pvr://recordings/deleted/radio/";
 
 CPVRRecordingsPath::CPVRRecordingsPath(const std::string &strPath)
 {
   std::string strVarPath(TrimSlashes(strPath));
   const std::vector<std::string> segments = URIUtils::SplitPath(strVarPath);
 
-  m_bValid  = ((segments.size() >= 3) && // at least pvr://recordings/[active|deleted]
+  m_bValid  = ((segments.size() >= 4) && // at least pvr://recordings/[active|deleted]/[tv|radio]
```

Comment:
should we switch active/deleted with tv/radio because iirc we have the same path syntax for channels `pvr://channels/[tv|radio]/groupname/`